indieweb-press-this.php

```diff
@@ -49,8 +49,8 @@ public static function bookmarklets_page() {
 			One big <a href="https://indieweb.org/">IndieWeb</a> <em>raison d’être</em> is using your own
 			web site to <a href="https://indieweb.org/reply">reply</a>,
 			<a href="https://indieweb.org/like">like</a>, <a href="https://indieweb.org/repost">repost</a>,
-			and <a href="https://indieweb.org/rsvp">RSVP</a> to posts and events. You do this by
-			annotating links on your site with simple
+            <a href="https://indieweb.org/follow">follow</a>, and <a href="https://indieweb.org/rsvp">RSVP</a>
```

Comment:
There seems to be something different with the indents

Reply:
tabs vs spaces, whee 😆